src/app/redux/uc.reducers.ts

```diff
@@ -169,28 +208,28 @@ function changeOrder(state: IUCAppState, action: UCTableOrderAction): IUCAppStat
 }
 
 function updateElements(state: IUCAppState): IUCAppState {
-    state.currentChanged = false;
-    state = filterElements(state, state.criterias);
-    state = sortElements(state);
-    if (!state.currentChanged) {
-        return state;
+    if (state.currentChanged || !state.currentSaved) {
+        putStateIntoURL(state);
+        state.currentSaved = true;
     }
-    putStateIntoURL(state);
     return state;
 }
 
 function initSettings(state: IUCAppState): IUCAppState {
     // Set elements settings
     const elementNames: Array<string> = [];
     const elementsEnabled: Array<boolean> = [];
-    DataService.data.forEach(value => {
```

Comment:
DataService.data.forEach((value, index) => {...

https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
also solved, github just doesn't notice for some reason...